lib/srv/db/cloud/iam.go

```diff
@@ -313,7 +321,11 @@ func (c *IAM) processTask(ctx context.Context, task iamTask) error {
 	if task.isSetup {
 		iamAuthErr := configurator.setupIAMAuth(ctx)
 		iamPolicySetup, iamPolicyErr := configurator.setupIAMPolicy(ctx)
-		c.iamPolicyStatus.Store(task.database.GetName(), iamPolicySetup)
+		statusEnum := types.IAMPolicyStatus_IAM_POLICY_STATUS_INVALID
```

Comment:
Doesn't "invalid" and "error" basically represent the same thing? Can we simplify and keep just one of these?

Reply:
I think you're right. I can combine them.